src/main/resources/tknsnippets.json

```diff
@@ -0,0 +1,279 @@
+{
+  "Tekton: ClusterTask": {
+    "prefix": "Tekton: ClusterTask",
+    "body": [
+      "apiVersion: tekton.dev/v1alpha1",
+      "kind: ClusterTask",
+      "metadata:",
+      "  name: ${1:foo}",
+      "  namespace: ${2: namespace}",
```

Comment:
Remove the namespace attribute. It is not required when creating a resource and will simplify the code

Reply:
I thought about this for some time and i added it to keep the **save** consistent with the **update**. When updating, before to push the tekton configuration to the cluster we validate the config by checking some field such as metadata name/namespace and spec. In the current state the **save** works the same. If we remove it we have to pass the namespace through the userData in the virtualFile and we end up having two "problems". 
* First, if the user wants to push to the cluster a tekton resource he previously created outside IJ he can't push it to the cluster without clicking on "new resource" and copy/paste the new content. (In the current situation he can open the file and ctrl+s will work the same and this is also a new feature we can add. When the user click on create "new pipeline" he can choose to open it from the filesystem or create it from scratch).
* Second, I don't think we are going to simplify the code by adding a new userData